vue/components/ui/molecules/button-upload/button-upload.vue

```diff
@@ -57,16 +57,12 @@ export const ButtonUpload = {
     props: {
         buttonText: {
             type: String,
-            default: "Upload Files"
+            default: null
         },
         disabled: {
             type: Boolean,
             default: false
         },
-        multiple: {

```

Comment:
Why remove? You're using it below:

![image](https://user-images.githubusercontent.com/3048875/132840693-7358c45d-eb98-4ddb-8ac5-7c361e35c7bd.png)

It seems to make sense to allow using the button to pick multiple files.

Reply:
This was moved to the mixin